### ISSUE TITLE
Export collection instance

### ISSUE THREAD

**Issue by pafurijaz** [KhronosGroup/glTF-Blender-IO#1389]
**Is your feature request related to a problem? Please describe.**
The problem seem solved by looking in the list (https://github.com/KhronosGroup/glTF-Blender-IO/pull/950)
But in the latest Blender Build the instanced collections from an other scene isn't exported into file.

**Describe the solution you'd like**
I have an other scene where I make 3d objects to be used as an instance in the main scene that I usually use, to be exported.

**Describe alternatives you've considered**
For now I use the OBJ exporter that support also instanced collections from other Blender file.

**Additional context**

**Comment by h33p** [comment]
For me it seems to be a problem that if a collection is disabled, its instances will not be exported.

If you link (not append) a (enabled) collection from another blender file, and make sure "Instance Collections" is selected, that collection will be exported fine.

However, then there is an additional issue where if you have linked collections inside a disabled collection, instances of the disabled collection will export linked collections, but will still skip the non-linked data.

For me, this behaviour seems inconsistent at best. If I have free time I'll see if I could fix it in the exporter code, but here are reproducable blender files:
[bug.zip](https://github.com/KhronosGroup/glTF-Blender-IO/files/6447373/bug.zip)

**Comment by h33p** [comment]
In addition, having the collection disabled will result in the linked object being exported. Not only that, the instance of it either doesn't export, or doesn't have transform applied for the linked collection.
[bug3.zip](https://github.com/KhronosGroup/glTF-Blender-IO/files/6447398/bug3.zip)

**Comment by pafurijaz** [comment]
I made an example of the case and it doesn't works
here in this file there are two scenes with obj instanced in the main
[Istance.tar.gz](https://github.com/KhronosGroup/glTF-Blender-IO/files/6447431/Istance.tar.gz)

**Comment by julienduroure** [comment]
All these 3 examples will be fixed in PR #1378

**Comment by pafurijaz** [comment]
Fantastic this news thanks

**Comment by pafurijaz** [comment]
Always related to instances, I found another type that is not converted and it is definitely worth considering this specific case and it is the instances on objects, which I personally use to avoid deformations on the chains as in this example, I hope you can also add this exception soon, and I await the implementation of this as soon as possible, thanks.
[link to Blender Manual Instancing](https://docs.blender.org/manual/en/latest/scene_layout/object/properties/instancing/index.html)

and here the screenshot of Blender view port and exported scene below
![image](https://user-images.githubusercontent.com/11751383/130333634-2e5933d6-3bcf-466d-b7b7-20c7bd515a4a.png)
![image](https://user-images.githubusercontent.com/11751383/130333682-5ea6427d-57d7-4d82-9507-22f65aa2f37f.png)

and this is the same scene exported with obj exporter..
![image](https://user-images.githubusercontent.com/11751383/130333763-f81cc176-8715-46ff-8090-9fbe1371a940.png)

**Comment by julienduroure** [comment]
@pafurijaz Let's keep this ticket about collection instance, and open a new one for duplifaces and duplivertices

**Comment by pafurijaz** [comment]
OK, but I see you did so are we alright?

**Comment by julienduroure** [comment]
Yes, sorry, it was not clear. I didn't ask you to open a new ticket yourself, but informed you that I opened a new one for this dupliface case.
Thanks !

**Comment by julienduroure** [comment]
Fixed by 69746b2b31eeee080d5dfd3c74f1e02742795ed2
